zstacklib/zstacklib/network/ovs/utils.py

```
@@ -0,0 +1,309 @@
+def probe_module(module_name: str) -> None:
+    """Load a kernel module.
+
+    Args:
+        module_name: Module name to load.
+
+    Raises:
+        OvsError: If module cannot be loaded.
+    """
+    ret = shell.run(f'modprobe {module_name}')
+    if ret != 0:
+        raise OvsError(f'Can not find module:{module_name}.')
```

Ruff (0.15.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)